pkg/proxy/ipvs/proxier.go

```diff
@@ -445,6 +444,9 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 		glog.Errorf("Failed to read file %s with error %v", kernelVersionFile, err)
 	}
 	kernelVersion := strings.TrimSpace(string(b))
+	if checkMinVersion(kernelVersion, "4.19") {
+		ipvsModules[4] = "nf_conntrack"
```

Comment:
The hardcoded index seems brittle, in case the `ipvsModules` initializer ever changes.

Modifying `ipvsModules` in place doesn't seem ideal either, but I don't know k8s coding standards.